tests/virt_autotest/reboot_and_wait_up_upgrade.pm

```diff
@@ -48,4 +48,18 @@ sub run {
     $self->reboot_and_wait_up($timeout);
 }
 
+sub enable_y2log_analysis {
```

Comment:
Can you give it a more direct name? eg upload_y2logs?

When we plan this task in scrum meeting, we plan it for two situations, right?  y2logs is not only needed by prj2, but also prj1 when guest installation fails, y2logs from the guest will be needed, right? Please put it under lib so that both prjs can use it.

Reply:
> Can you give it a more direct name? eg upload_y2logs?
sure, will change name as upload_y2logs here. 

> When we plan this task in scrum meeting, we plan it for two situations, right? 
yes, but this PR will deal with trello card as below:
https://trello.com/c/B6N0w909/562-p2-offline-host-guest-upgrade-does-not-provide-log-collectiony2log-after-failing-to-reach-reboot-step-it-never-supports-this-but

>y2logs is not only needed by prj2, but also prj1 when guest installation fails, y2logs from the guest will be needed, right? 
the other PR will fix up this y2logs for prj1 and prj4
PR#737 [Log Enhancement] enable y2log for guest installation and offline guest upgrade
https://github.com/SUSE/qa-automation/pull/737

>  Please put it under lib so that both prjs can use it.

Reply:
Ah, great. 

About putting upload_y2logs into lib, I think it is still a right choice. It is a utility function, for which lib is suitable for.

Reply:
sure, got it.

Reply:
had moved upload_y2logs func into lib file virt_autotest::utils

Reply:
DONE

Reply:
DONE? Why current code still has the old enable_y2log_analysis function here?

Reply:
had used pload_y2logs already. 
please check it again.